packages/core/test/ServerTest.ts

```diff
@@ -115,6 +119,32 @@ class WoTServerTest {
         expect(thing).to.have.property("properties").to.have.property("myProp");
     }
 
+    @test async "should be able to destroy a Thing based on a thingId"() {
+        let desc = `{
+            "@context": ["https://w3c.github.io/wot/w3c-wot-td-context.jsonld"],
+            "@type": ["Thing"],
+            "title": "myDestroyThing",
+            "id": "1234567",
+            "properties": {
+                "myProp" : {
+                }
+            }
+        }`;
+        let thing = await WoTServerTest.WoT.produce(JSON.parse(desc));
+        expect(thing).to.exist;
+        // test TD
+        expect(thing.getThingDescription()).to.have.property("title").to.equal("myDestroyThing");
+        expect(thing.getThingDescription()).to.have.property("id").to.equal("1234567");
+        // test presence (and destroy)
+        if (WoTServerTest.servient.getThing("1234567") == null) {
```

Comment:
Why don't use `expect` here? Fore example: `expect(WoTServerTest.servient.getThing("1234567")).to.not.be.null`

Reply:
makes sense